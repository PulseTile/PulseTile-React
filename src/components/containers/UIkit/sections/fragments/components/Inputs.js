import React from 'react';

/**
 * This component returns content of Inputs section
 *
 * @return {XML}
 * @constructor
 */
const Inputs = () => {
    return (
        <div id="inputs" className="ui-section">
            <strong className="ui-title">Inputs</strong>
            <div className="form-group-wrapper">
                <div className="row">
                    <div className="col-xs-12 col-sm-6">
                        <div className="form-group">
                            <label for="name" className="control-label">Input simple</label>
                            <div className="input-holder">
                                <input type="text" className="form-control input-sm" />
                            </div>
                        </div>
                    </div>
                    <div className="col-xs-12 col-sm-6">
                        <div className="form-group">
                            <label for="name" className="control-label">Input simple (disabled)</label>
                            <div className="input-holder">
                                <input type="text" className="form-control input-sm" value="text" disabled/>
                            </div>
                        </div>
                    </div>
                </div>
                <div className="row">
                    <div className="col-xs-12 col-sm-6">
                        <div className="form-group">
                            <label className="control-label">Input with addon</label>
                            <div className="inner-addon addon-left">
                                <div className="addon">
                                    <i className="fa fa-filter"></i>
                                </div>
                                <input type="text" className="form-control input-sm"/>
                            </div>
                        </div>
                    </div>
                    <div className="col-xs-12 col-sm-6">
                        <div className="form-group">
                            <label className="control-label">Input with addon (disabled)</label>
                            <div className="inner-addon addon-left">
                                <div className="addon">
                                    <i className="fa fa-filter"></i>
                                </div>
                                <input type="text" className="form-control input-sm" value="text" disabled/>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    )
};

export default Inputs;
