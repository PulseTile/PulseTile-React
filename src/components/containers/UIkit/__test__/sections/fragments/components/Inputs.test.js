import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Inputs from '../../../../sections/fragments/components/Inputs';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <Inputs />', () => {
  it('should renders Inputs with props correctly', () => {
    const component = shallow(<Inputs />);
    expect(component).toMatchSnapshot();
  });
});
