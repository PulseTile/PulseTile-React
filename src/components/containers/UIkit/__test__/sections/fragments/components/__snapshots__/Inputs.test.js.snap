// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Inputs /> should renders Inputs with props correctly 1`] = `
<div
  className="ui-section"
  id="inputs"
>
  <strong
    className="ui-title"
  >
    Inputs
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="row"
    >
      <div
        className="col-xs-12 col-sm-6"
      >
        <div
          className="form-group"
        >
          <label
            className="control-label"
            for="name"
          >
            Input simple
          </label>
          <div
            className="input-holder"
          >
            <input
              className="form-control input-sm"
              type="text"
            />
          </div>
        </div>
      </div>
      <div
        className="col-xs-12 col-sm-6"
      >
        <div
          className="form-group"
        >
          <label
            className="control-label"
            for="name"
          >
            Input simple (disabled)
          </label>
          <div
            className="input-holder"
          >
            <input
              className="form-control input-sm"
              disabled={true}
              type="text"
              value="text"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-xs-12 col-sm-6"
      >
        <div
          className="form-group"
        >
          <label
            className="control-label"
          >
            Input with addon
          </label>
          <div
            className="inner-addon addon-left"
          >
            <div
              className="addon"
            >
              <i
                className="fa fa-filter"
              />
            </div>
            <input
              className="form-control input-sm"
              type="text"
            />
          </div>
        </div>
      </div>
      <div
        className="col-xs-12 col-sm-6"
      >
        <div
          className="form-group"
        >
          <label
            className="control-label"
          >
            Input with addon (disabled)
          </label>
          <div
            className="inner-addon addon-left"
          >
            <div
              className="addon"
            >
              <i
                className="fa fa-filter"
              />
            </div>
            <input
              className="form-control input-sm"
              disabled={true}
              type="text"
              value="text"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
